etl_project/pipelines/w.py

```diff
@@ -0,0 +1,72 @@
+from dotenv import load_dotenv
+import os
+from etl_project.connectors.postgresql import *
+from etl_project.assets.extract_news import *
+from sqlalchemy import *
+
+
+if __name__ == "__main__":
+    loads = load_dotenv()
+    API_KEY = os.environ.get("API_KEY")
+    DB_USERNAME = os.environ.get("DB_USERNAME")
+    DB_PASSWORD = os.environ.get("DB_PASSWORD")
+    SERVER_NAME = os.environ.get("SERVER_NAME")
+    DATABASE_NAME = os.environ.get("DATABASE_NAME")
+    PORT = os.environ.get("PORT")
+
+    # API_KEY = os.getenv("API_KEY")
```

Comment:
not the biggest deal, but why the commented out environment variables right below the environment variables?